app-text/multitail/multitail-7.0.0.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake flag-o-matic optfeature toolchain-funcs
+
+DESCRIPTION="Tail with multiple windows"
+HOMEPAGE="http://www.vanheusden.com/multitail/ https://github.com/folkertvanheusden/multitail/"
+SRC_URI="https://github.com/folkertvanheusden/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux"
+IUSE="debug examples unicode"
+RESTRICT="test" # bug 492270
```

Comment:
I won't block merging on this but maybe revisit it / patch out the static analysis stuff if it has meaningful tests apart from that? (I don't know if it does, not looked)

Reply:
It's good that you say it, 7.0.0 says:
`No tests found. Skipping.`

So, I guess if there are no tests no more, we don't need to restrict.

Reply:
Correct!